packages/compartment-mapper/src/import-hook.js

```diff
@@ -726,11 +727,9 @@ export function makeImportNowHookMaker(
       }
 
       if (exitModuleImportNowHook) {
-        // this hook is responsible for ensuring that the moduleSpecifier actually refers to an exit module
-        const exitRecord = exitModuleImportNowHook(
-          moduleSpecifier,
-          packageLocation,
-        );
+        // This hook is responsible for ensuring that the moduleSpecifier
+        // actually refers to an exit module.
+        const exitRecord = exitModuleImportNowHook(moduleSpecifier);
```

Comment:
Bugfix revealed by typing.

Reply:
@boneskull I’ve posted a new commit to this PR that fixes the typing bug but not as reflected in this outdated comment. It turns out that `exitModuleImportHook` does require the second argument but did not provide it in all cases. So, the new commit includes the second argument in the hook type and threads it through the various call sites. Please take a look.